### PR TITLE
Create new versioned release (R24.08)

### PR DESCRIPTION
**Pull request by jSchuetz88**
This PR creates a new version for R24.08. 

This PR does not create or add any new content:
- The docusaurus versioning plugin was executed and the contents of the KITs were copied into a versioned folder for R24.08.
- Additionally, the version folder for R24.03 has been deleted.

As all the contents have already been approved in the previous smaller PRs, we still need an approval by eclipse cause of the huge number of changes.

[IP Issue](https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/15836) already created

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [ ] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Review by maximilianong — APPROVED** [review]

**Comment by stephanbcbauer** [comment]
@eclipse-tractusx/automotive-tractusx-committers [IP Issue](https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/15836) is approved, but there is still the discussion label. I am in contact with them if this is a mistake.

Could you, please have a look at this PR? We will merge it, after we will get the approval by Eclipse.

**Comment by stephanbcbauer** [comment]
> @eclipse-tractusx/automotive-tractusx-committers [IP Issue](https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/15836) is approved, but there is still the discussion label. I am in contact with them if this is a mistake.
> 
> Could you, please have a look at this PR? We will merge it, after we will get the approval by Eclipse.

"discussion label" removed, please have a look at this PR @eclipse-tractusx/automotive-tractusx-committers

**Review by danielmiehle — APPROVED** [review]
LGTM

**Pull request merged**
